conans/client/build/cmake.py

```diff
@@ -275,6 +277,13 @@ def install(self, args=None, build_dir=None):
     def test(self, args=None, build_dir=None, target=None, output_on_failure=False):
         if not self._conanfile.should_test:
             return
+        if cross_building(self._conanfile.settings):  
```

Comment:
Is it possible that someone has test functionality in their CMakeLists that is able to run their test in some kind of emulator/virtualbox? Maybe able to run 32bits apps in 64bits environment (like Windows)?

Just wondering if this should be explicitly disabled by users in the recipes, not internally, or at least possible to opt-in/out this behavior with an argument to the ``cmake.test()`` method.